src/Services/PaymentCreation.php

```diff
@@ -235,6 +249,26 @@ public function capturePayment(Payment $payment, Order $order)
         $payment->amount = $orderData['amounts'][0]['grossTotal'];
         $payment->receivedAt = date('Y-m-d H:i:s');
 
+        try {
+            /** @var CurrencyExchangeRepositoryContract $currencyService */
+            $currencyService = pluginApp(CurrencyExchangeRepositoryContract::class);
+
+            $defaultCurrency = $currencyService->getDefaultCurrency();
+
+            if ($payment->currency != $defaultCurrency) {
+                $payment->exchangeRatio = $currencyService->getExchangeRatioByCurrency($payment->currency);
+                $payment->amount = round(
+                    $currencyService->convertToDefaultCurrency(
+                        $payment->currency,
+                        $payment->amount,
+                        $payment->exchangeRatio
+                    ),
+                    2);
```

Comment:
Why is here still the currency convertion? And the code on top looks wrong to me: `$payment->amount = $orderData['amounts'][0]['grossTotal'];`. Maybe lets talk after the daily about this.